ideascube/serveradmin/package.py

```diff
@@ -36,7 +47,7 @@ def from_csv(cls, csv_path):
             except csv.Error:
                 dialect = csv.unix_dialect()
             medias = list(csv.DictReader(content.splitlines(), dialect=dialect))
-            return cls(working_dir, medias)
+            return cls(working_dir, medias, report)
```

Comment:
The same old discussion here. For me they are not a keyword arguments. They are arguments with default value.